src/common/legacy/src/graph_transformer.cpp

```diff
@@ -219,6 +219,7 @@ static std::vector<std::string> skipConstInfer = {
     "CumSum",       // Const inference function for CumSum is not implemented
     "Convolution",  // Const inference function for Convolution is not implemented
     "Eltwise",      // Const inference function for Eltwise is not implemented
+    "Copy",
```

Comment:
> consider "GnaCopy" to be more unique

I would like to keep it as is for several reasons:

1. Operation is defined in `ov::intel_gna::op` namespace. `GNA` prefix in operation name looks extra.
2. For legacy code we already have handling of `Cop`y layer. Adding `GnaCopy` in addition to `Copy` would lead to possible mess.
3. PWL operation is introduced already inside plugin without any prefix.

Reply:
The new class is scoped fine, but `skipConstInfer` does not contain class reference but pure string.
maybe `skipConstInfer` could be refactored to check against fully scoped namespace too.

Reply:
> The new class is scoped fine, but `skipConstInfer` does not contain class reference but pure string. maybe `skipConstInfer` could be refactored to check against fully scoped namespace too.

This legacy code should be removed soon. I would like to avoing speding time for refactoring legacy code,